libfwupdplugin/fu-usb-device-ms-ds20.c

```diff
@@ -34,8 +34,8 @@ fu_usb_device_os20_type_to_string(guint16 type)
 		return "subset-header-configuration";
 	if (type == USB_OS_20_SUBSET_HEADER_FUNCTION)
 		return "subset-header-function";
-	if (type == USB_OS_20_FEATURE_COMPATBLE_ID)
-		return "feature-compatble-id";
+	if (type == USB_OS_20_FEATURE_COMPATIBLE_ID)
+		return "feature-compatible-id";
```

Comment:
This change is more interesting. As something might care about the returned string.

Reply:
I think this instance it's fine to change